recipe/patch_yaml/openff-qcsubmit-qcelemental.yaml

```diff
@@ -0,0 +1,11 @@
+# This is another Pydantic v2 patch, one of our upstreams
+# (QCElemental) is undergoing their v1 -> v2 migration which will
+# likely result in API breaks
+if:
+  subdir_in: noarch
+  name: openff-qcsubmit-base
+  timestamp_lt: 1728073334
```

Comment:
The timestamp looks too short? The example in the README uses milliseconds.

Reply:
That was it - thanks @scaramallion!